image/discourse_test/Dockerfile

```
@@ -17,23 +17,24 @@ RUN chown -R discourse . &&\
 
 FROM base AS with_browsers
 
-RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add - &&\
-    echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list &&\
-    apt update &&\
-    apt install -y libgconf-2-4 libxss1 google-chrome-stable firefox-esr &&\
+ADD install-chrome /tmp/install-chrome
+RUN /tmp/install-chrome
+
+RUN apt update &&\
+    apt install -y libgconf-2-4 libxss1 firefox-esr &&\
```

keeping layers to minimum

```suggestion
RUN /tmp/install-chrome &&\
    apt update &&\
    apt install -y libgconf-2-4 libxss1 firefox-esr &&\
```